code/modules/projectiles/guncrafting/attachments/energy_bullets.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/weapon/gun_attachment/energy_bullet
+	gun_type = CUSTOMIZABLE_ENERGY
+	uses_overlay = FALSE
+	icon_state = "bullet"
+	not_okay = /obj/item/weapon/gun_attachment/energy_bullet
+
+/obj/item/weapon/gun_attachment/energy_bullet/disorienter
+	name = "Energy Disorienter"
+	desc = "Drowsiness, less damage."
+
+/obj/item/weapon/gun_attachment/energy_bullet/disorienter/on_fire(var/obj/item/weapon/gun/owner, var/obj/item/projectile/bullet)
+	..()
+	bullet.drowsy += 10
+	if(bullet.damage >= 5)
+		bullet.damage -= 5
+	else
+		bullet.damage = 0
+
+/obj/item/weapon/gun_attachment/energy_bullet/decloner
+	name = "Energy Decloner"
+	desc = "Force your enemies into cryogenics.."
+
+/obj/item/weapon/gun_attachment/energy_bullet/decloner/on_fire(var/obj/item/weapon/gun/owner, var/obj/item/projectile/bullet)
+	..()
+	bullet.damage_type = CLONE
+	bullet.damage *= 0.7
+	bullet.irradiate = 10
+
+/obj/item/weapon/gun_attachment/energy_bullet/speed
+	name = "Energy Speeder"
+	desc = "Beep Beep."
+
+/obj/item/weapon/gun_attachment/energy_bullet/speed/on_fire(var/obj/item/weapon/gun/owner, var/obj/item/projectile/bullet)
+	..()
+	bullet.speed *= 0.5
+	bullet.damage *= 0.5
+	bullet.spread += 0.3
```

Comment:
I recommend replacing its spread with an increased power consumption, due to how fast the projectiles are going. This is more of a balance concern.

Also, since it seems like the size mod was removed, I recommend you re-add it, and also give it an increased energy consumption. Both should have an energy consumption of x2.